org.gnome.Papers.json

```diff
@@ -39,6 +39,25 @@
         "*.a"
     ],
     "modules": [
+        {
+            "name": "gdk-pixbuf",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dbuiltin_loaders=jpeg,png,bmp",
+                "-Dgtk_doc=true",
+                "-Dman=false"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gdk-pixbuf/2.42/gdk-pixbuf-2.42.9.tar.xz",
```

Comment:
2.42.12 is the latest one it seems and what we have in the runtime, does that one built or are there issues with it?